suite-native/config/src/supportedNetworks.ts

```diff
@@ -1,43 +1,68 @@
 import { A } from '@mobily/ts-belt';
 
+import { isTestnet } from '@suite-common/wallet-utils';
 import { Network, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
 
 const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol'];
```

Comment:
Isn't this blacklist redundant now? Since `sol` neither `dsol` are not `networkSymbolsWhitelistMap`, there is no need for filtering them out by matching this blacklist.